tool/site/client/src/services/test-packs/communicationRequests.ts

```diff
@@ -0,0 +1,41 @@
+import * as fhir from "fhir/r4"
+
+export function createCommunicationRequest(patientEntry: fhir.BundleEntry) : fhir.BundleEntry {
+  return {
+    fullUrl: "urn:uuid:51793ac0-112f-46c7-a891-9af8cefb206e",
+    resource: {
+      resourceType: "CommunicationRequest",
+      status: "unknown",
+      subject: {
+        reference: "urn:uuid:78d3c2eb-009e-4ec8-a358-b042954aa9b2"
+      },
+      payload: [
+        {
+          contentString: "TEST PRESCRIPTION - DO NOT DISPENSE"
+        }
+      ],
+      requester: {
+        type: "Organization",
+        identifier: {
+          system: "https://fhir.nhs.uk/Id/ods-organization-code",
+          value: "RBA"
+        }
+      },
+      recipient: [
+        {
+          type: "Patient",
+          identifier: {
+            system: "https://fhir.nhs.uk/Id/nhs-number",
+            value: getNhsNumber(patientEntry)
+          }
+        }
+      ]
+    } as fhir.CommunicationRequest
```

Comment:
just wondering why we need this casting here? are our underlying types not quite right?

Reply:
It's the fhir types here, bundle.entry is a derived fhirResource type so could be any number of a lot of resource types, using this to add additional type safety to our resource building